src/setStateIndicator/setIsSick.test.js

```diff
@@ -0,0 +1,37 @@
+import { setIsSick } from "./setIsSick";
+
+jest.useFakeTimers();
+
+describe("setIsSick", () => {
+  it("calls callback", () => {
+    const callback = jest.fn();
+    jest.spyOn(global.Math, "random").mockReturnValue(0.1);
+    setIsSick(callback);
+
+    jest.advanceTimersByTime(5000);
+
+    expect(callback).toBeCalled();
+    expect(callback).toHaveBeenCalledTimes(1);
```

Comment:
toBeCalled is a less reliable version of toHaveBeenCalledTimes. toBeCalled would also return true if your function was called 2 or 3 times, which is not what we want. Try to be as specific as possible in your `expect` statements

Reply:
ok